upgrade/upgrade.go

```diff
@@ -91,8 +92,18 @@ func Upgrade(p *project.Project, from, to string, opts UpgradeOpts) error {
 		return err
 	}
 
-	if opts.Wait {
-		return rFromService.Wait(service)
+	if opts.Wait || opts.Clean {
+		err := rFromService.Wait(service)
```

Comment:
This can be just 
```go
if err := rFromService.Wait(service); err != nil {
     return err
}
```

Same comment on L103.  We should enable go lint/vet on this project so that it will just report these nit-picky comments.